### PR TITLE
Feature autoredirect users with one organization

### PR DESCRIPTION
**Pull request by julian-berbel**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -26,6 +27,17 @@ class ApplicationController < ActionController::Base
                 :theme_stylesheet_url,
                 :extension_javascript_url
 
+  def should_redirect?
```
Comment (flbulgarelli) [inline app/controllers/application_controller.rb]: Please improve name to `should_redirect_to_main_organization?`

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -26,6 +27,17 @@ class ApplicationController < ActionController::Base
...
+    should_choose_organization? && current_user.accessible_organizations.length == 1
+  end
+
+  def redirect_to_main_organization!
+    labo = Mumukit::Platform.laboratory
```
Comment (flbulgarelli) [inline app/controllers/application_controller.rb]: Please avoid using shortnames.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -26,6 +27,17 @@ class ApplicationController < ActionController::Base
...
+    flash.notice = t :you_were_redirected, url: view_context.link_to(t(:here), labo.organic_url(Organization.central))
+
+    redirect_to labo.organic_url(current_user.accessible_organizations.first)
```
Comment (flbulgarelli) [inline app/controllers/application_controller.rb]: Please delegate to `current_user.main_organization`

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -26,6 +27,17 @@ class ApplicationController < ActionController::Base
                 :theme_stylesheet_url,
                 :extension_javascript_url
 
+  def should_redirect?
+    should_choose_organization? && current_user.accessible_organizations.length == 1
```
Comment (flbulgarelli) [inline app/controllers/application_controller.rb]: Please delegate to `current_user.has_main_organization?`

Comment (flbulgarelli) [reply app/controllers/application_controller.rb]: That way you can remove logic from controller, move it to model and test it easily.

**Comment by julian-berbel** [comment]
Uploading an image here on request of @flbulgarelli 
image

File: lib/mumuki/laboratory/locales/es.yml [hunk 5]
```diff
@@ -187,5 +187,6 @@ es:
   working: "Procesando"
   wrong_answer: La respuesta no es correcta
   you_must_sign_in_before_submitting: Tenés que iniciar sesión antes de empezar a enviar tus soluciones
+  you_were_redirected: Fuiste redirigido a tu organización. Si te querés quedar en central, hace click %{url}
```
Comment (flbulgarelli) [inline lib/mumuki/laboratory/locales/es.yml]: Te redirigimos

**Pull request merged**